pyproject.toml

```
@@ -38,6 +30,16 @@ cu12 = [
     "nvidia-cuda-nvrtc-cu12",
     "nvidia-nvjitlink-cu12"
 ]
+cu13 = [
+    "cuda-bindings==13.0.*",
+    "cuda-core==0.3.*",
+    "cuda-python==13.0.*",  # supports all CTK 13.x
```

I think we can pin this to just `13.*` since we're guaranteeing not to have API / ABI breaking changes?

```suggestion
    "cuda-python==13.*",
```